STARTERKIT/components/init/_colors.scss

```diff
@@ -22,9 +22,20 @@ $chroma: add-colors((
   grey-extra-light:       ('black' lighten 93.33%), // #eee
   white:                  #fff,
```

Comment:
https://github.com/JohnAlbin/chroma/blob/master/README.md

  // Define a primary highlight color that has the value of our "blue" color.
  // Note: if blue was specified without quotes, Chroma would interpret that as
  // the color keyword blue and not a reference to the "blue" color name.